tests/integration/test_interaction.py

```diff
@@ -284,6 +295,62 @@ def get_locations():
             self.assertDictEqual(locations_after_reset[item_key], initial_locations[item_key])
             self.assert_reverted_item(item_key)
 
+    def alt_text_image_displayed(self, items_map):
+        #Verify that the alt text for the underlying image appears when the user navigates to it with the keyboard.
+        target_img = self._page.find_element_by_css_selector('.target-img')
+        alt_text = target_img.get_attribute('alt')
+
+        # Navigate to dropzone using keyboard
+        items_container = self._page.find_element_by_css_selector('.target')
+
+        # Check that text that appears is the same as the alt-text
+        self.assertEqual(items_container.get_attribute('aria-describedby'), alt_text)
+
+    def alt_text_keyboard_controls(self, items_map):
+        for item_key, definition in items_map.items():
+            item = self._get_unplaced_item_by_value(definition.item_id)
+            ActionChains(self.browser).move_to_element(item).perform()
+            keyboard_help_text = u'Press "Enter", "Space", "Ctrl-m", or "⌘-m" on an item to select it for dropping, then navigate to the zone you want to drop it on.'
+
+            self.assertEqual(item.get_attribute('aria-describedby'), keyboard_help_text)
+
+    def alt_text_for_occupied_zones(self, items_map, scroll_down=100, action_key=None):
+        #Verify that the alt text for the zone displays if there is already an object in it.
+        popup = self._get_popup()
+        feedback_popup_content = self._get_popup_content()
+
+        # Scroll drop zones into view to make sure Selenium can successfully drop items
+        self.scroll_down(pixels=scroll_down)
+
+        #place all items in zones where they belong
+        for definition in self._get_items_with_zone(items_map).values():
```

Comment:
@mtyaka It complains when it sees N identical lines in two (three, four, M) places. we probably should update that XBlock too :)